### PR TITLE
Project Information in json. Temperature statistics by mast and by turbine 

### PR DESCRIPTION
**Pull request by jsrodrigo**
This pull request includes the following changes:
- `Project Information` tab in the excel format was missing in the json format. 
- `Temperature` tab in the excel format has been formulated similarly to the `Frequency` tab with frequency and number of samples for measurement device and turbine sites.

### CODE REVIEW

**Comment by NievesBurillo** [comment]
Hi Javier, 
Thanks for the help. 
It looks good to me :)
Hope the rest can also comment and we can have this closed by early next week.

File: README.md [hunk 1]
```diff
@@ -12,17 +12,15 @@ This framework provides for organization of wind and other climatic condition pa
 Furthermore, the framework presents a Digital Exchange Format, consisting of complementary excel and JSON files, embodying the above mentioned reporting requirements and data structure. The excel and JSON files contain identical information. The excel file is useful for human readability, while JSON format is better suited for machine readability. The Digital Exchange Format contains the minimum requirements of data for assessment of the suitability of a wind project development site for any contemplated turbine make/model, and together with the report provides a clear view of project meteorological conditions
 
 ## Usage
-The .xlsx file and Javasript Object Notation (JSON) file are intended to faciliate and streamline data exchange between wind industry stakeholders.  The IEC team has also included a versions that have been populated with example data. 
+The .xlsx file and Javasript Object Notation (JSON) file are intended to faciliate and streamline data exchange between wind industry stakeholders.  The IEC team has also included a versions that have been populated with example data.
```
Comment (Dynorat) [inline README.md:15]: Should read as " The IEC team has also included a version that has been populated with example data."

File: site_suitability_input_conditions_form_v16_20240305_with_example_data.json [hunk 2]
```diff
@@ -251,7 +265,7 @@
             "Associated Data Source": "Gobblers Knob East",
             "Ve50": 59.36000000000001,
             "V50": 42.400000000000006,
-            "CoV": "N/A",
+            "COV": "N/A",
```
Comment (Dynorat) [inline site_suitability_input_conditions_form_v16_20240305_with_example_data.json:268]: is this correct?  I think "CoV" had a specific scientific meaning.  @NievesBurillo ? @youngli-jp

**Review by Dynorat — CHANGES_REQUESTED** [review]
some basic comments here to be double checked.  otherwise looks good to me.  We can double check before the final version release once the repo is a bit cleaner

**Review by Dynorat — APPROVED** [review]
makes sense to me

**Review by Dynorat — COMMENTED** [review]
suggest adding any inline comments that might help the user understand the python script and usage.  Perhaps this is in the readme somewhere? @jsrodrigo @youngli-jp  thoguhts?

File: site_suitability_input_conditions_form_v16_20240305_with_example_data.json (hunk 2 above)
Comment (Dynorat) [reply site_suitability_input_conditions_form_v16_20240305_with_example_data.json:268]: #10 linking the related issue here for resolution

**Comment by jonssonchristian** [comment]
Hi @jsrodrigo, @NievesBurillo, @Dynorat and @youngli-jp, 

I see this pull request has not been merged yet. Is there anything holding it up? If everyone is happy with the changes, can we go ahead and merge it? If there are further refinements planned, perhaps those can be covered in a separate pull request.

**Comment by NievesBurillo** [comment]
Thanks for following up on this topic Christian.

From my side it is OK to proceed, if no one has comments, I actually thought it was already done.

I do not know how to do it so someone else will have to.
With best regards,
Nieves Burillo
Tel.: +45 30376316

**Comment by jonssonchristian** [comment]
Hi @NievesBurillo,

Thank you very much for the feedback. I am happy to complete the merge if everyone is happy with that. I will leave a little bit of time in case anyone else has comments.

**Comment by jonssonchristian** [comment]
Since there were no further comments, I go ahead and merge this pull request now.

**Pull request merged**